src/circuit_breaker.erl

```diff
@@ -196,9 +197,17 @@ clear(Service) ->
 %% @doc Print information regarding services.
 %% @end
 info() ->
-  format("Service", "Status", "Error", "Timeout", "CallTimeout"),
-  format(dup(31, $-), dup(15, $-), dup(8, $-), dup(8, $-), dup(8, $-)),
-  ets:foldl(fun do_info/2, ok, ?TABLE).
+  Text = info_to_text(),
+  io:put_chars(Text).
+
+-spec info_to_text() -> io_lib:chars().
+%% @doc Print information regarding services.
+%% @end
+info_to_text() ->
+  Header  = format("Service", "Status", "Error", "Timeout", "CallTimeout"),
+  Split   = format(dup(31, $-), dup(15, $-), dup(8, $-), dup(8, $-), dup(8, $-)),
+  Entries = ets:foldr(fun do_info/2, [], ?TABLE),
```

Comment:
The ets table isn't `ordered_set` so calling `foldr/3` doesn't have the effect implied by its name. Please change that back to `foldl/3`.
Please also squash to a single commit.